api/src/main/java/org/pmiops/workbench/db/model/User.java

```diff
@@ -130,4 +131,15 @@ public void setFirstSignInTime(Timestamp firstSignInTime) {
   public void setAuthorities(Set<Authority> newAuthorities) {
     this.authorities = newAuthorities;
   }
+
+  @OneToMany(fetch = FetchType.EAGER, mappedBy = "user")
+  public Set<WorkspaceUserRole> getWorkspaceUserRoles() {
+    return workspaceUserRoles;
+  }
+
+  // Necessary for Spring initialization of the object.
+  // Not actually supported because it won't delete old entries.
```

Comment:
Use /** comments instead of //. That way IDE / Javadoc will include it. We may even want to mark this as @Deprecated.